### PR TITLE
chore(package): update `loader-utils` v0.2.15...1.1.0

### LINKED ISSUE #137: [Bug] DOT files are always treated as directories
DOT files are always treated as directories.

{from: path.resolve(__dirname, 'application/admin/.htaccess'), to: 'admin/.htaccess', toType: 'file'}

will result in a folder /admin/.htaccess/ being created when a file /admin/.htaccess should be created. If the "to" is changed to something like 'admin/zzz.htaccess' then it creates it as a file. It appears that the "toType" is ignored if it has a leading dot.

### PR DESCRIPTION
**Pull request by evilebottnawi**
### `Noteable Changes`

- Some code was removed because he is fixed problems in old `loader-utils`.
- Use `absolute` path and `context` to right determinate interpolate name.

### `Issues`

- Fixes #137

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "fs-extra": "^4.0.2",
     "glob": "^7.1.2",
     "is-glob": "^4.0.0",
-    "loader-utils": "^0.2.15",
+    "loader-utils": "^1.1.0",
```
Comment (michael-ciniawsky) [inline package.json]: This is strictly a 🏷  Major 😞 . We cpuld do it, if we find consensus about being a bit looser with semver here, since node =< 4 is EOL anyways 😛

File: src/writeFile.js [hunk 2]
```diff
@@ -31,37 +31,20 @@ export default function writeFile(globalRef, pattern, file) {
             if (pattern.toType === 'template') {
```
Comment (michael-ciniawsky) [inline src/writeFile.js]: Please split into separate PR, bc of the comment above

File: package.json (hunk 1 above)
Comment (evilebottnawi) [reply package.json]: @michael-ciniawsky i think it is more fix then breaking changes because some interpolation don't works as expected

File: src/writeFile.js (hunk 2 above)
Comment (evilebottnawi) [reply src/writeFile.js]: @michael-ciniawsky after upgrade this code do error, need remove this lines, that everything worked as it should

File: package.json (hunk 1 above)
Comment (evilebottnawi) [reply package.json]: And yes all that < node 4 is EOL :smile:

File: src/writeFile.js (hunk 2 above)
Comment (michael-ciniawsky) [reply src/writeFile.js]: So the dependency upgrade is required for these changes ? What exactly errors atm (I'm not too familiar with this package yet tbh 😅 )

File: package.json (hunk 1 above)
Comment (michael-ciniawsky) [reply package.json]: I'm basically ok with it, but let's wait a bit what others say

File: src/writeFile.js (hunk 2 above)
Comment (evilebottnawi) [reply src/writeFile.js]: @michael-ciniawsky yes with .dot files and files without directory `foo.txt`

Comment (michael-ciniawsky) [reply src/writeFile.js]: #137 ? If see that one, but the issue originates from `loaderUtils.interpolateName()` ?

Comment (evilebottnawi) [reply src/writeFile.js]: @michael-ciniawsky this problems avoid after upgrade

**Comment by d3viant0ne** [comment]
I've updated the target branch to `next` & we can land this along with #146. For simplicities sake we can land this one first so you don't have to deal with the defaults rebase & i'll pick up the changes over in #146

File: package.json (hunk 1 above)
Comment (d3viant0ne) [reply package.json]: How about we skip the grey area on the breaking nature of loader-utils & just roll it in with defaults which is most definitely a major. We can publish to `@beta / @next / @purple_fluffie_bunnies` off the `#next` branch until everyone is satisfied with it. Also gives me a chance to test against angular/cli without blocking other PRs in master

**Comment by michael-ciniawsky** [comment]
@evilebottnawi Could you rebase please, I set the `engines` field in `package.json` to `node >= v4.0.0` a while ago and there weren't any complaints so far. `node =< v4.0.0` is EOL for over a year now I think we can land this as a patch in this case :) ?

**Comment by evilebottnawi** [comment]
@michael-ciniawsky I hope we will not break anything :smile:
